spec/dummy/config/initializers/devise.rb

```diff
@@ -1,10 +1,24 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  # config.secret_key = '2955fa4bb843d7a6735f5e6c42eb3381a767ad77f5cd37f4903d64eba1873caa66bd5abfee5ac733c255eae9f8a356111389cb7468c61d144b4914f7d7decf47'
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '5895b2e3c90f41c90d489bb3d91b666cb710dd91e2fa737be7fdaa90233f581b5870b4834d9df003e17dc1acd6113fe090cbe12be6cb6293a0e1a01b7fb19cde'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/LineLength: Line is too long. [154/100]